internal/pkg/service/common/etcdop/key.go

```diff
@@ -56,8 +56,8 @@ func (v Key) Exists(client etcd.KV, opts ...etcd.OpOption) op.BoolOp {
 	)
 }
 
-func (v Key) Get(client etcd.KV, opts ...etcd.OpOption) op.GetOneOp {
-	return op.NewGetOneOp(
+func (v Key) Get(client etcd.KV, opts ...etcd.OpOption) op.WithResult[*op.KeyValue] {
+	return op.NewForType(
 		client,
 		func(_ context.Context) (etcd.Op, error) {
 			return etcd.OpGet(v.Key(), opts...), nil
```

Comment:
I removed old `op.GetOneOp` types, we have now the `WithResult` generic type for it. 

This is `Get` operation on untyped key, there is only one `Get` method.
Bellow are `Get*` opeations on TYPED key.